src/navigate-pop/navigate-files-folders.md

```diff
@@ -58,6 +58,9 @@ Enter a search term; matched items will automatically populate below the search
 
 Click the down arrow and enter a date in `When` to limit the search to files' creation date. You can also select a relative date range from the list. Choose to apply the date query to `Last Modified`, `Last Used`, or `Created` file attributes.
 
+![Change file path]
+In order to type in a file path like `/etc/` hit the `Ctrl` and `L` key. The search bar will then show the absolute file path and you'll be able to enter a new one.
```

Comment:
> In order to type in a file path like `/etc/` hit the `Ctrl` and `L` key.

A comma is needed before "hit". (I'm also not sure if we should use the word "hit" rather than something else like "type" or "press.")

> The search bar will then show the absolute file path and you'll be able to enter a new one.

A comma is needed before "and".